desk/src/pages/desk/ticket/editor/BottomSection.vue

```diff
@@ -140,6 +142,19 @@ const insertRes = createResource({
 	onSuccess() {
 		clean();
 	},
+	validate(params) {
+		if (isEmpty(params.doc.content)) {
+			return "Message is empty";
+		}
+	},
+	onError(error) {
+		createToast({
+			title: "Error sending reply",
+			text: error.messages?.join("\n"),
```

Comment:
Error from `validate()` will be available as an `Error` object. You can get the message from this with `error.message`